### PR TITLE
Invalid argument

### PR DESCRIPTION
**Pull request by Cromel**

### CODE REVIEW

File: api.go [hunk 1]
```diff
@@ -28,6 +28,11 @@ type responseError struct {
 // easyjson:json
 type errorDetails struct {
 	ErrorCode errorCode `json:"errorCode,omitempty"`
+	FieldViolations []fieldValidation `json:"field_violations"`
```
Comment (mkorolyov) [inline api.go]: inconsistensy in naming: field`Violations` vs field`Validation`

File: go.mod [hunk 2]
```diff
@@ -1,6 +1,6 @@
-module github.com/humans-group/fcm
+module github.com/humans-group/go-fcm
 
-go 1.17
+go 1.19
 
 require (
 	github.com/gojuno/minimock/v3 v3.0.6
```
Comment (aleksey-korolev) [inline go.mod]: Let's update minimock to v3.3.6

File: api.go (hunk 1 above)
Comment (Cromel) [reply api.go]: Thx for noticing. Renamed

File: go.mod (hunk 2 above)
Comment (Cromel) [reply go.mod]: done

**Review by mkorolyov — APPROVED** [review]

**Review by aleksey-korolev — APPROVED** [review]

**Pull request merged**